ee/cis/win-10/cis-policy-queries.yml

```diff
@@ -4770,27 +4752,27 @@ spec:
   query: |
     SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Services\\Tcpip\\Parameters\EnableICMPRedirect' AND data = 0);
   purpose: Informational
-  tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_18.4.5, CIS_group_policy_template_required
+  tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_2.0.0, CIS_bullet_18.5.5, CIS_group_policy_template_required
   contributors: rachelelysia
 ---
 apiVersion: v1
 kind: policy
 spec:
   name: >
-    CIS - Ensure 'MSS: (KeepAliveTime) How often keep-alive packets are sent in milliseconds' is set to 'Enabled: 300,000 or 5 minutes'
+    CIS - Ensure 'LSA Protection' is set to 'Enabled'
```

Comment:
This item index is 2.0.0 -> 18.4.6
But since this area is now 18.5.x , this policy should be moved up.